src/main/java/org/gridsuite/shortcircuit/server/repositories/ShortCircuitAnalysisResultRepository.java

```diff
@@ -302,4 +309,23 @@ public String findStatus(UUID resultUuid) {
             return null;
         }
     }
+
+    private Pageable addDefaultSort(Pageable pageable, String defaultSortColumn) {
+        if (pageable.isPaged()) {
```

Comment:
Maybe it's the "defaultSort" naming which is misleading ?
It's not really a default sort because even if some sort are defined it will be added, it's only here to allow deterministic order when querying paged resources, which is for me critically needed.

When querying not paged resources, Do we need a deterministic order as well ? considering ORDER BY has a performance cost ? Not sure...

Reply:
alright @sBouzols fair enough :) By the way about performances in DB it's good to add a column that is ordered by in a composite index, I don't know if it's done or planned already.